x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/index.test.tsx

```diff
@@ -169,4 +171,28 @@ describe('event details footer component', () => {
     const element = wrapper.queryByTestId('side-panel-flyout-footer');
     expect(element).toBeNull();
   });
+
+  describe('Alerts', () => {
+    const propsWithAlertIndex = {
+      ...defaultProps,
+      expandedEvent: {
+        eventId: ecsData._id,
+        indexName: `.internal.${DEFAULT_ALERTS_INDEX}-testSpace`,
```

Comment:
Consider removing the trailing `.` after `internal` in this test:

```suggestion
        indexName: `.internal${DEFAULT_ALERTS_INDEX}-testSpace`,
```

,because (temporarily) adding the following before the `expect`: 

```ts
      console.log(
        '--> propsWithAlertIndex.expandedEvent.indexName is',
        propsWithAlertIndex.expandedEvent.indexName
      );
```

demonstrates that the additional trailing `.` after `internal` results in an index name with two-dots:

### Before (two dots after `internal`):
```
--> propsWithAlertIndex.expandedEvent.indexName is .internal..alerts-security.alerts-testSpace
```

### After (one dot after `internal`):

```
-> propsWithAlertIndex.expandedEvent.indexName is .internal.alerts-security.alerts-testSpace
```